change_notes/2023-10-25-a0-1-1.md

```diff
@@ -0,0 +1,4 @@
+ * `A0-1-1` - address a number of false positive issues:
+   * Exclude compiler-generated variables, such as those generated for range-based for loops.
+   * Exclude variables in uninstantiated templates, for which we have no precise data on uses.
+   * Deviations can now be applied to the useless assignment as well as the variable itself.
```

Comment:
```suggestion
   * Deviations can now be applied to the useless assignment as well as the variable itself.
```
With the removal of `isExcluded` on `v` it can only be applied to the useless assignment.

Reply:
Done.